version-control/git.md

```diff
@@ -1,13 +1,31 @@
 # Git
 
 Git is a tool that helps you manage versions of your code and work in collaboration with team members. There is a lot to know about it, but in this tutorial we will cover only basic usage.
-You can download it and follow the installation instructions here .
-When asked if you'd like to install git bash, say yes.
 
-Install Git
+## Install Git
+
 Click on the arrow (^) below to expand the section for your operating system.
 
 {% include "./git-windows.md" %}
 
 {% include "./git-mac.md" %}
 
+## Verify Install and Initialize User Settings (for both Windows and Mac)
+
+
+Next, we need to configure our Git version control so that you can use it with your GitHub account.
+
+Back in your terminal, set your user name by running the command:
```

Comment:
Might need to specify GitBash for windows users at this point. Or make a note somewhere that going forward (once they have Git installed) all command line operations will use GitBash.

Reply:
I added the initialization instructions to both git instructions, one specific to Mac & Linux, the other specific to Windows. I believe that addresses this comment.